### PR TITLE
Add command that lists all supported SB commands

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/commands/screenbuilder.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var allSupportedCommands = this.$screenBuilderService.allSupportedCommands().wait();
+			_.each(allSupportedCommands, (command, index) => this.$logger.out("%s: %s", index.toString(), command));
+		}).future<void>()();
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+}
+$injector.registerCommand("screenbuilder", ScreenBuilderCommand);
```
Comment (Fatme) [inline lib/commands/screenbuilder.ts]: I couldn't figure out better name for this command, so I'm open to any suggestion.

File: lib/commands/screenbuilder.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export class ScreenBuilderCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $screenBuilderService: IScreenBuilderService) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var allSupportedCommands = this.$screenBuilderService.allSupportedCommands().wait();
+			_.each(allSupportedCommands, (command, index) => this.$logger.out("%s: %s", index.toString(), command));
```
Comment (rosen-vladimirov) [inline lib/commands/screenbuilder.ts]: why do we need the index here? What's the value of showing it to the users?

**Comment by rosen-vladimirov** [comment]
run ci

File: lib/commands/screenbuilder.ts (hunk 2 above)
Comment (Fatme) [inline lib/commands/screenbuilder.ts]: I think it is better to be enumerated :smile_cat:

**Comment by teobugslayer** [comment]
We talked in person and decided to put the SB commands in the text of the help command.

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**